components/AssetTabContent/OverviewContent.tsx

```diff
@@ -31,8 +31,9 @@ const OverviewSubTitle: FunctionComponent<PropsWithChildren> = ({ children }) =>
 )
 
 export const OverviewContent = ({ content }: Props) => {
-  const usePortableText = config.USE_PORTABLE_TEXT
-
+  const usePortableText = process.env.NEXT_PUBLIC_USE_PORTABLE_TEXT === "true"
```

Comment:
why was this removed from `config.ts`?

Reply:
I left a comment about that one, it didn't update properly 🤷‍♀️  Was left with false server side and true client side, even after restarting the server ++

Reply:
that's peculiar 🤔